### ISSUE TITLE
Build fails with "can't find crate for `rustc_front`"

### ISSUE THREAD

**Issue by kc1212** [canndrew/error_def#1]
I'm getting the following error after running `cargo build`.

```
Compiling error_def v0.3.6 (file:///home/kc1212/rustcode/error_def)
src/lib.rs:7:1: 7:26 error: can't find crate for `rustc_front` [E0463]
src/lib.rs:7 extern crate rustc_front;
             ^~~~~~~~~~~~~~~~~~~~~~~~~
error: aborting due to previous error
error: Could not compile `error_def`.
```

I read somewhere that `rustc_front` comes with the compiler, so it may be related to the version of rust that I'm using - `rustc 1.10.0-nightly (8da2bcac5 2016-04-28)`.

**Comment by kc1212** [comment]
Looks like `rustc_front` got moved - https://github.com/rust-lang/rust/pull/32570

**Comment by canndrew** [comment]
Thanks for the bug report! This is fixed in 0.3.7
